tox.ini

```diff
@@ -1,7 +1,7 @@
 [tox]
 skipsdist=True
 skip_missing_interpreters = True
-envlist = lint, unit, integration
+envlist = lint, static, unit, integration
```

Comment:
```suggestion
envlist = lint, static, unit, integration, prettify
```

Should `prettify` be included here?

Reply:
envlist is what runs if you execute tox without arguments. so prettify should probably be excluded?

Reply:
Ah ofc, my bad.